src/main/java/org/opensearch/flowframework/workflow/AbstractCreatePipelineStep.java

```diff
@@ -63,7 +63,8 @@ public PlainActionFuture<WorkflowData> execute(
         WorkflowData currentNodeInputs,
         Map<String, WorkflowData> outputs,
         Map<String, String> previousNodeInputs,
-        Map<String, String> params
+        Map<String, String> params,
+        String tenantId
```

Comment:
Note this won't be used for this step, but the interface will require it so no harm doing this.